src/scripts/browserDeviceProfile.js

```diff
@@ -1250,4 +1266,3 @@ export default function (options) {
 
     return profile;
 }
-
```

Comment:
I would undo the removal of the last line. AFAIK, every file in the project has a caret return on the last line.

Reply:
Oh, it's a duplicate. Okay, then.